x-pack/solutions/observability/plugins/observability_onboarding/test/scout/ui/tests/onboarding_ui_validation.spec.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout-oblt';
+import { test } from '../fixtures';
+
+const TEST_TIMEOUT = 3 * 60 * 1000;
+
+test.describe('Onboarding UI Validation', () => {
+  test.describe.configure({
+    timeout: TEST_TIMEOUT,
+    retries: 2,
+  });
+
+  test.beforeEach(async ({ pageObjects, browserAuth }) => {
+    await browserAuth.loginAsAdmin();
+    await pageObjects.onboarding.goto();
+    await pageObjects.onboarding.waitForMainTilesToLoad();
+  });
+
+  test(
+    'should always show Host, Kubernetes, and Cloud tiles',
+    { tag: ['@ess', '@svlOblt'] },
+    async ({ pageObjects }) => {
+      await expect(pageObjects.onboarding.hostUseCaseTile).toBeVisible();
+      await expect(pageObjects.onboarding.kubernetesUseCaseTile).toBeVisible();
+      await expect(pageObjects.onboarding.cloudUseCaseTile).toBeVisible();
+    }
+  );
+
+  test(
+    'should show Application tile and complete mode copy when in complete deployment',
+    { tag: ['@ess', '@svlOblt'] },
+    async ({ pageObjects }) => {
+      test.skip(
+        !(await pageObjects.onboarding.applicationUseCaseTile.isVisible()),
+        'Application tile not visible - likely logs-essentials mode'
+      );
```

Comment:
This part doesn't feel right. We're assuming that Application tile is not visible because kibana is in logs-essentials mode, but we don't know this for sure, it could be hidden because of a bug or some new logic in the code. And at the same time we sort of "detect" logs-essentials mode by checking the Application tile visibility but this seems unreliable. 

Would be better to have an explicit flag inside the test that would tell which mode we're running in, have you maybe checked already if it's possible to pass the current product tier inside scout tests?